xmtp_db/src/encrypted_store/refresh_state.rs

```diff
@@ -203,7 +208,11 @@ impl<C: ConnectionExt> QueryRefreshState for DbConnection<C> {
         let mut cursor_map: HashMap<Vec<u8>, i64> = HashMap::new();
         for conversation_id in conversation_ids {
             let cursor = self
-                .get_last_cursor_for_id(conversation_id, EntityKind::CommitLogDownload)
+                .get_last_cursor_for_id(
+                    conversation_id,
+                    EntityKind::CommitLogDownload,
+                    xmtp_configuration::Originators::REMOTE_COMMIT_LOG.into(),
```

Comment:
need to create an issue for remote commit log originator, for now 10 should work